src/iop/exposure.c

```diff
@@ -723,7 +723,7 @@ static void autoexp_callback(GtkWidget *button, gpointer user_data)
 
   if(self->request_color_pick == DT_REQUEST_COLORPICK_MODULE)
   {
-    dt_lib_colorpicker_set_area(darktable.lib, 0.99);
+    dt_lib_colorpicker_set(darktable.lib, 0.99, 0);
```

Comment:
But here, if the point picker is selected, where will it be placed?
I didn't try it, but this may be confusing.